docs/ExportFormats.md

```diff
@@ -0,0 +1,26 @@
+# Export Formats
+
+## Summary table CSV export
+
+The summary table CSV export includes the following columns:
+
+* `Phyloreference ID`: A URL or URI (absolute, relative or local) that identifies this phyloreference.
+  * Example `#Alligatoridae`
+* `Label`: A label used to identify this phyloreference.
+  * Example: `Alligatoridae`
+* `Type`: The type of this phyloreference.
+  * May be one of the following values:
+    * `Apomorphy-based clade definition`
+    * `Maximum clade definition`
+    * `Minimum clade definition`
+    * `Invalid definition (...)` (with the reason for the invalid definition in brackets)
+* `Definition`: The clade definition as entered in the `Definition in free-form text`.
+  * Example: `Last common ancestor of Crocodylus niloticus, Osteolaemus tetraspis, and Tomistoma schlegelii and all of its descendents.`
+* `Internal specifier N`: A description of each internal specifier (_N_ is a number starting from 1).
+  * Example: `Caiman crocodilus`
+* `External specifier N`: A description of each external specifier (_N_ is a number starting from 1).
+  * Example: `Caiman crocodilus`
+* `Phylogeny: [Phylogeny label] expected`: The node resolved for this phylogeny (_N_ is a number starting from 1, and _Phylogeny label_ is the label of this phylogeny).
```

Comment:
```suggestion
* `Phylogeny: [Phylogeny label] expected`: The node in this phylogeny to which the phyloreference was expected to resolve (by the author) (_Phylogeny label_ is the label of this phylogeny).
```

Reply:
There is no _N_ (nor should there be, right?), so I deleted the part about explaining _N_ (potentially a leftover from overzealous copy&paste?)

Reply:
And are you sure this should be the label of the phylogeny and not the node in the phylogeny?